cmd/utils.go

```diff
@@ -6,11 +6,14 @@ import (
 	"strings"
 	"time"
 
+	"github.com/hashicorp/hcl/v2"
 	"github.com/hashicorp/hcl/v2/hclsimple"
 	"github.com/mvisonneau/go-helpers/logger"
 	"github.com/urfave/cli"
+	"github.com/zclconf/go-cty/cty/function"
 
 	tfcw "github.com/mvisonneau/tfcw/lib/client"
+	"github.com/mvisonneau/tfcw/lib/funcs"
```

Comment:
do you mind if we use `functions` instead ?

Reply:
Yup sure 🙂 done!